can-route-pushstate.js

```diff
@@ -287,12 +294,18 @@ canReflect.assignSymbols(PushstateObservable.prototype, {
 	"can.valueHasDependencies": function() {
 		return true;
 	},
-	//!steal-remove-start
-	"can.getName": function() {
-		return "PushstateObservable<" + this.value + ">";
-	},
-	//!steal-remove-end
-});
+};
+
+//!steal-remove-start
+if (process.env.NODE_ENV !== 'production') {
+	pushstateObservableProto[canSymbol.for("can.getName")] = function() {
```

Comment:
canSymbol.for() isn't needed because these are turned into symbols by assignSymbols() below.